Yafc/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -1144,7 +1157,9 @@ private void BuildGoodsIcon(ImGui gui, Goods? goods, ProductionLink? link, float
             && ((dropdownType == ProductDropdownType.Fuel && recipe.fixedFuel)
             || (dropdownType == ProductDropdownType.Ingredient && recipe.fixedIngredient == goods)
             || (dropdownType == ProductDropdownType.Product && recipe.fixedProduct == goods))) {
-            evt = gui.BuildFactorioObjectWithEditableAmount(goods, displayAmount, ButtonDisplayStyle.ProductionTableScaled(iconColor), tooltipOptions: tooltipOptions);
+
```

Comment:
There is an empty line here (AFAIK we don't have empty lines at block beginnings?)

Reply:
The empty line fits there thematically for the same reason why the [empty-line suggestion](https://github.com/shpaass/yafc-ce/blob/master/Docs/CodeStyle.md?plain=1#L54) exists for the declarations. It allows to better understand where the condition ends and the body starts.